helpdesk_mgmt/models/helpdesk_ticket_team.py

```diff
@@ -18,7 +20,14 @@ class HelpdeskTeam(models.Model):
         default=lambda self: self.env['res.company']._company_default_get(
             'helpdesk.ticket')
     )
-
+    user_id = fields.Many2one('res.users', string='Team Leader',
+                              check_company=True)
```

Comment:
Why `check_company=True`?

Reply:
Like the sale team reference, to make sure the team/user coherence company:
https://github.com/OCA/OCB/blob/13.0/addons/sales_team/models/crm_team.py#L51

Reply:
I usually see this as a problem, as a user can belong to several companies and this prevents it if the user is logged in another one.

Reply:
I'll leave the field for another pr, so we'll check it out